config/settings.yml

```diff
@@ -1283,6 +1283,7 @@
         :poll: 10.seconds
       :event_catcher_swift:
         :poll: 10.seconds
+      :event_catcher_openstack_service: "auto"
```

Comment:
K, we'll just need to make sure this gets picked up when OSP is extracted to its own repo /cc @durandom